modules/packages/ZMQ.chpl

```diff
@@ -165,13 +165,52 @@ versions may expose more features native to ZeroMQ.
 Serialization
 +++++++++++++
 
-In Chapel, sending or receiving messages on a :record:`Socket` uses
-`multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
-and the :chpl:mod:`Reflection` module to serialize primitive and user-defined
-data types whenever possible.  Currently, the ZMQ module serializes primitive
-numeric types, strings, and records composed of these types.  Strings are
-encoded as a length (as `int`) followed by the character array
-(in bytes).
+In Chapel, sending or receiving messages is supported for a variety of types.
+Primitive numeric types and strings are supported as the foundation.
+In addition, user-defined :type:`record` types may be serialized automatically
+as `multipart messages <http://zguide.zeromq.org/page:all#Multipart-Messages>`_
+by internal use of the :chpl:mod:`Reflection` module.
+Currently, the ZMQ module can serialize records of primitive numeric types,
+strings, and other serializable records.
+
+.. note::
```

Comment:
(nitpick) Do you think this note would be as effective without referencing "Chapel 1.16"? I don't see any references to Chapel versions elsewhere in module documentation, presumably because the documentation is versioned itself and user-facing changes are usually handled with deprecation messages.